### PR TITLE
Adding a qa environment settings file and simplifying the way setting…

### PR DESCRIPTION
**Pull request by cdmo**
…s are loaded (no more case)

### CODE REVIEW

File: lib/traject/psulib_config.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 $LOAD_PATH << File.expand_path('../', __dir__)
 
 ENV['RUBY_ENVIRONMENT'] = 'dev' if ENV['RUBY_ENVIRONMENT'].nil?
-
+puts ENV['RUBY_ENVIRONMENT']
```
Comment (banukutlu) [inline lib/traject/psulib_config.rb]: @cdmo this is not intentional, right ;)

Comment (cdmo) [reply lib/traject/psulib_config.rb]: lol no.

Comment (cdmo) [inline lib/traject/psulib_config.rb]:
```suggestion
```

Comment (cdmo) [reply lib/traject/psulib_config.rb]:
```suggestion
```

File: lib/traject/psulib_config.rb [hunk 2]
```diff
@@ -49,7 +44,7 @@
     provide 'marc4j_reader.permissive', indexer_settings['marc4j_reader_permissive']
     provide 'marc4j_reader.source_encoding', indexer_settings['marc4j_reader_source_encoding']
     # defaults to 1 less than the number of processors detected on your machine
-    provide 'processing_thread_pool', indexer_settings['processing_thread_pool']
+    provide 'processing_thread_pool', indexer_settings['processing_thread_pool'].to_i
```
Comment (banukutlu) [inline lib/traject/psulib_config.rb]: 👌

**Pull request merged**